README.md

```diff
@@ -156,6 +156,7 @@ which will automatically pick up the config file.
 | CLAMAV_BASE                 | ClaMAV base uRL                       | True     |
 | CLAMAV_PORT                 | CLAMAV port                           | True     |
 | NODE_TLS_REJECT_UNAUTHORIZED| Ignore TLS verification               | True     |
+| ENABLE_UNAVAILABLE_PAGE     | Show service unavailable page         | False    |
 
```

Comment:
Tried locally and it works great. Just one minor comment that please update README to have a section that if this flag is made to true, then service unavailable page will be shown.